.solhint.json

```diff
@@ -29,7 +43,10 @@
     "imports-on-top": "error",
```

Comment:
We could use the new rule introduced in v5.0.3 to order imports: https://github.com/protofire/solhint/releases/tag/v5.0.3

Reply:
Done